internal/grpcwrap/provider.go

```diff
@@ -73,6 +74,7 @@ func (p *provider) GetSchema(_ context.Context, req *tfplugin5.GetProviderSchema
 			Block:   convert.ConfigSchemaToProto(dat.Body),
 		}
 	}
+	// Do we add state stores here? Why aren't ephemeral resources here?
```

Comment:
This confused me, and I'll take another look tomorrow

Reply:
Added in 81b44cf

I think ephemeral resources were omitted here accidentally, and that didn't matter because it's a testing resource for E2E tests.